src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -2482,6 +2482,9 @@ namespace openstudio{
           unsigned month = sqlite3_column_int(sqlStmtPtr, b++);
           unsigned day = sqlite3_column_int(sqlStmtPtr, b++);
           unsigned intervalMinutes = sqlite3_column_int(sqlStmtPtr, b++); // used for run periods
+          if (reportingFrequency == ReportingFrequency::Annual){
+            intervalMinutes = 8760; // used for annual periods
+          }
```

Comment:
@joseph-robertson 365 days = 8760 hours

1. This is in hours, should be in minutes, so 8760*24
2. What happens for a leap year?

Reply:
@jmarrec Good questions. I need to go back and remember why I made these changes/updates.